.travis.yml

```diff
@@ -21,3 +21,13 @@ cache:
 # This makes the cache smaller and helps highlight problems with a clean build
 before_cache:
  - find $HOME/.m2/repository/org/sakaiproject -name \*-SNAPSHOT.\*  -delete
+
+# connect SonarCloud
+addons:
+  sonarcloud:
+    organization: "sakai-sandbox"
+    token:
+    secure: "G34ivrx9eXat+i535OfNFtrokRxNF8UAPWVP6sKb8XoRTflhKcuR/WzhX0n11gI7iloNDEzKGFmG5x5NmcuC1EqqRf/N+DiW6l7c0LAcvKcSaJXKlvtSb2kAdywLa4VJnrvQYZIzETNhWoY/uY329coXtaWxk170/YWKnp5DHE8="
```

Comment:
This is safe to be here right? It looks like we could define a hidden environment variable in the build too.

Reply:
build env var is the way to go

Reply:
This is encrypted but can be a build environment variable if preferred. Though I am unsure if the build env var goes in encrypted?

Reply:
Set a build var. Not sure why this is failing to push as i have an identical setup for https://github.com/steveswinsburg/sakai-archiver - will keep working on it.